api/projects/projects-router.js

```diff
@@ -1 +1,59 @@
 // Write your "projects" router here!
+const express = require('express')
+const Projects = require(`./projects-model`)
+//middleware:
+const { validateProjectId, validateProjects } = require(`./projects-middleware`)
+
+const router = express.Router()
+
+
+router.get(`/projects`, (req, res, next) => {
+   Projects.get()
+      .then(action => {
+         res.status(200).json(action)
+      })
+      .catch(next)
```

Comment:
Same as previous assignment: I know that the course you're following is teaching the catch error at the end of everything. Personally I think having the catch error sent within the catch of the response is better bc then you can add a message to it to be able to debug the code better later on.